backport_packages/setup_backport_packages.py

```diff
@@ -143,6 +139,22 @@ def run(self):
 
 
 def change_import_paths_to_deprecated():
+    from bowler import LN, TOKEN, BowlerTool, Capture, Filename, Query
+    from fissix.pytree import Leaf
+
+    BowlerTool.IN_PROCESS = True
```

Comment:
I think we can remove it, now we know that the problem was the `for` loop not multiprocessing 
```suggestion

```